javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/bootstrap/internal/DeprecatedConfigProperties.java

```diff
@@ -1,50 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package io.opentelemetry.javaagent.bootstrap.internal;
-
-import static java.util.Collections.emptyList;
-import static java.util.logging.Level.WARNING;
-
-import java.util.List;
-import java.util.logging.Logger;
-
-/**
- * This class is internal and is hence not for public use. Its APIs are unstable and can change at
- * any time.
- */
-public final class DeprecatedConfigProperties {
```

Comment:
wonder if we should keep this class around for next time we rename a config property 😬

Reply:
Impossible! LOL. I can add it back, but it'll just be a stub. Is that better than re-adding it when needed? Without references would future us even find it?

Reply:
idk, let's see what @laurit thinks

Reply:
I think it could be useful we still have a bunch of experimental properties that will probably renamed in the future. One place where it could be used immediately is
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/8885012bcb6fd8da16291fd2edb3f5cc6bb0cfcb/instrumentation/kafka/kafka-clients/kafka-clients-0.11/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/kafkaclients/v0_11/metrics/KafkaMetricsUtil.java#L19-L21

Reply:
Added a PR for that kafka property https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/10354

Reply:
Makes sense then, I'll put it back. ❤️